stdlib/int64.md

```diff
@@ -0,0 +1,833 @@
+No other changes yet.
+</details>
+
+```grain
+sub : (Int64, Int64) -> Int64
+```
+
+Computes the difference of its operands.
+
+Parameters:
+
+|param|type|description|
+|-----|----|-----------|
+|`x`|`Int64`|The first operand|
+|`y`|`Int64`|The second operand|
+
+Returns:
+
+|type|description|
+|----|-----------|
+|`Int64`|The difference of the two operands|
+
+### Int64.**mul**
+
+<details disabled>
+<summary tabindex="-1">Added in <code>0.2.0</code></summary>
+No other changes yet.
+</details>
+
+```grain
+mul : (Int64, Int64) -> Int64
+```
+
+Computes the product of its operands.
+
+Parameters:
+
+|param|type|description|
+|-----|----|-----------|
+|`x`|`Int64`|The first operand|
+|`y`|`Int64`|The second operand|
+
+Returns:
+
+|type|description|
+|----|-----------|
+|`Int64`|The product of the two operands|
+
+### Int64.**div**
+
+<details disabled>
+<summary tabindex="-1">Added in <code>0.2.0</code></summary>
+No other changes yet.
+</details>
+
+```grain
+div : (Int64, Int64) -> Int64
+```
+
+Computes the quotient of its operands using signed division.
+
+Parameters:
+
+|param|type|description|
+|-----|----|-----------|
+|`x`|`Int64`|The first operand|
+|`y`|`Int64`|The second operand|
+
+Returns:
+
+|type|description|
+|----|-----------|
+|`Int64`|The quotient of its operands|
+
+### Int64.**divU**
+
+<details disabled>
+<summary tabindex="-1">Added in <code>0.2.0</code></summary>
+No other changes yet.
+</details>
+
+```grain
+divU : (Int64, Int64) -> Int64
+```
+
+Computes the quotient of its operands using unsigned division.
+
+Parameters:
+
+|param|type|description|
+|-----|----|-----------|
+|`x`|`Int64`|The first operand|
+|`y`|`Int64`|The second operand|
+
+Returns:
+
+|type|description|
+|----|-----------|
+|`Int64`|The quotient of its operands|
+
+### Int64.**rem**
+
+<details disabled>
+<summary tabindex="-1">Added in <code>0.2.0</code></summary>
+No other changes yet.
+</details>
+
+```grain
+rem : (Int64, Int64) -> Int64
+```
+
+Computes the remainder of the division of its operands using signed division.
+
+Parameters:
+
+|param|type|description|
+|-----|----|-----------|
+|`x`|`Int64`|The first operand|
+|`y`|`Int64`|The second operand|
+
+Returns:
+
+|type|description|
+|----|-----------|
+|`Int64`|The remainder of its operands|
+
+### Int64.**remU**
+
+<details disabled>
+<summary tabindex="-1">Added in <code>0.2.0</code></summary>
+No other changes yet.
+</details>
+
+```grain
+remU : (Int64, Int64) -> Int64
+```
+
+Computes the remainder of the division of its operands using unsigned division.
+
+Parameters:
+
+|param|type|description|
+|-----|----|-----------|
+|`x`|`Int64`|The first operand|
+|`y`|`Int64`|The second operand|
+
+Returns:
+
+|type|description|
+|----|-----------|
+|`Int64`|The remainder of its operands|
+
+### Int64.**mod**
+
+<details disabled>
+<summary tabindex="-1">Added in <code>0.2.0</code></summary>
+No other changes yet.
+</details>
+
+```grain
+( mod ) : (Int64, Int64) -> Int64
```

Comment:
@ospencer any idea why this is wrapped in parens? Something held over from ocaml's oprint?

Reply:
PR up that removes these: https://github.com/grain-lang/grain/pull/1104